### PR TITLE
CIV-7846 - Added Logic for Evidence Upload LiP notification

### PR DESCRIPTION
**Pull request by m-meulendijks-v1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-7846

### Change description ###
Added a reference to the LiP template, and logic that allows an email to be sent to a LiP instead of an LR.
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — DISMISSED** [review]

File: commons-civil/src/main/java/uk/gov/hmcts/reform/civil/notify/NotificationsProperties.java
Comment (miguelMolina3691) [inline commons-civil/src/main/java/uk/gov/hmcts/reform/civil/notify/NotificationsProperties.java:1]: Shouldn't this be in civil-commons?

**Review by miguelMolina3691 — CHANGES_REQUESTED** [review]
NotificationsProperties.java is in civil-commons

**Review by miguelMolina3691 — DISMISSED** [review]

File: commons-civil/src/main/java/uk/gov/hmcts/reform/civil/notify/NotificationsProperties.java
Comment (m-meulendijks-v1) [reply commons-civil/src/main/java/uk/gov/hmcts/reform/civil/notify/NotificationsProperties.java:1]: It is, but it is also here from an earlier attempt, I think. Removed it now.

**Review by LeonardoPalmeiro — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -284,6 +285,7 @@ notifications:
   notifyRespondentLiPMediationSuccessfulTemplate: "38600a69-c147-451c-a0ea-40c212fda8bc"
   notifyRespondentLiPMediationSuccessfulTemplateWelsh: "aaa19fef-2b16-42fa-a1b9-5af699fcaa58"
 
+
```
Comment (hmcts-version1-ignacio) [inline src/main/resources/application.yaml:288]: Extra blank line?

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by maciejmodzelewski — DISMISSED** [review]

**Review by maciejmodzelewski — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by kurt-uk — APPROVED** [review]

**Pull request merged**